openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -269,6 +269,76 @@ paths:
         '500':
           description: Internal Server Error
       x-spring-paginated: true
+  /debt-position-types/receipts:
+    get:
+      tags:
+        - Receipts
+      summary: "Retrieve a page of receipts based on filters"
+      operationId: getReceipts
+      parameters:
+        - name: organizationId
+          in: query
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: receiptOrigin
+          in: query
+          required: true
+          schema:
+            type: 'ReceiptOriginType'
+        - name: operatorExternalUserId
+          in: query
+          required: true
+          schema:
+            type: string
```

Comment:
don't ask for the operator!
This is not to be asked to the client! this is an implicit filter added by BFF to perform authorization on the request